.github/workflows/docker-publish.yml

```diff
@@ -19,18 +19,27 @@ jobs:
         run: docker build . --tag $IMAGE_PATH/$IMAGE_NAME:latest
         
       - name: Test image
+        timeout-minutes: 25
         run: |
           docker run $IMAGE_PATH/$IMAGE_NAME:latest /bin/bash -c "
           git clone https://github.com/Open-MSS/MSS.git &&
           cd MSS &&
           git checkout develop &&
           source /opt/conda/bin/activate mssenv &&
-          conda install --file requirements.d/development.txt &&
+          mamba install --file requirements.d/development.txt &&
           apt-get update &&
           apt-get install xvfb -y &&
-          conda install pyvirtualdisplay &&
+          mamba install pyvirtualdisplay &&
           [ -d "~/.cache/matplotlib" ] && rm -r ~/.cache/matplotlib/ ;
           pytest mslib"
```

Comment:
because mamba is not shell script we have all env vars available only after our own restart of the env.